src/Bowling.cpp

```diff
@@ -64,3 +65,37 @@ void Bowling::readDirectory(path p)
         std::cout << ex.what() << '\n';
     }
 }
+
+std::string Bowling::toString() const
+{
+    std::stringstream ss;
+    std::string data;
+    for (int i = 0; i < getNumberOfLanes(); i++)
+    {
+        Lane lane = getLane(0);
+        std::string laneData;
+        std::stringstream no; 
+        no << ++i;
```

Comment:
In my opinion you want do `(i+1)`, not increment variable which is used in for loop.